SVGLayer.h

```diff
@@ -8,12 +8,76 @@
 
 #import <QuartzCore/QuartzCore.h>
 
-/// An SVG-backed CALayer that you can view that parses, renders, and display an SVG file.
+/// A CALayer subclass that renders an SVG file.
 
 @interface SVGLayer : CALayer
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*!
+ *  @discussion Renders the provided SVG.
+ *
+ *  @param aSVGName The name of an SVG file in the app bundle.
+ *
+ *  @code // iOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.renderSVGNamed("tiger")
+ view.layer.addSublayer(myLayer)
+
+ // macOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.geometryFlipped = true
+ myLayer.renderSVGNamed("tiger")
+ view.layer?.addSublayer(myLayer)
+ *
+ */
+- (void)renderSVGNamed:(NSString *)aName;
```

Comment:
This is a misleading name. Rendering happens multiple times after the image is loaded (every time layer frame etc) and is handled by the shape layers. This renaming does not make things more understandable.